client/go/internal/cli/cmd/visit.go

```diff
@@ -118,9 +126,22 @@ $ vespa visit --content-cluster search # get documents from cluster named "searc
 	cmd.Flags().BoolVar(&vArgs.makeFeed, "make-feed", false, `output JSON array suitable for vespa-feeder`)
 	cmd.Flags().BoolVar(&vArgs.pretty, "pretty-json", false, `format pretty JSON`)
 	cmd.Flags().IntVar(&vArgs.chunkCount, "chunk-count", 1000, `chunk by count`)
+	cmd.Flags().Uint64Var(&vArgs.from, "from", 0, `Timestamp to visit from, in microseconds`)
```

Comment:
Good points! I also do not like the microseconds, I think seconds since UTC is what people are used to / expect? I cannot see a use case for visiting using microsecond ranges ...

Reply:
A microsecond unit was used for the Doc V1 API for two reasons:

1. Documents are assigned (pseudo-)microsecond timestamps in the backend
2. It's what `vespa-visit` exposes in its parameters, and the other Doc V1 parameters mirror its semantics

There is nothing wrong with using a more "friendly" unit in the `vespa` CLI since it's a higher level tool. Seconds since UTC epoch is the most obvious candidate.

Reply:
your suggestion "--to and --from, but accept either RFC3339 format or seconds since UTC" sounds good to me, but we can probably keep your --from-timestamp etc as-is for now